lib/unlocked/EditTile.dart

```diff
@@ -0,0 +1,485 @@
+                                    )
+                                  },
+                                  child: Image.asset(
+                                    widget.tile.imageUrl,
+                                    width: screenSize.width / 2,
+                                  ),
+                                ),
+                              ),
+                            ),
+                            Container(
+                              height: 24,
+                              child: Text(widget.tile.name,
+                                  style: TextStyle(
+                                    fontSize: 12.0,
+                                    fontFamily: "Robotto",
+                                    fontWeight: FontWeight.w500,
+                                  )),
+                            ),
+                          ],
+                        ),
+                      ),
+                    ),
+                  ),
+                ),
+              ],
+            ),
+            Card(
+              elevation: 0.0,
+              // margin: const EdgeInsets.fromLTRB(1.0, 8.0, 1.0, 8.0),
+              child: Column(
+                children: <Widget>[
+                  Container(
+                    decoration: BoxDecoration(
+                      border: Border(
+                        bottom: new BorderSide(color: Colors.black12),
+                      ),
+                    ),
+                    child: Column(
+                      children: [
+                        ListTile(
+                          // Tile Name
+                          title: Text('Tile Name',
+                              style:
+                                  TextStyle(fontSize: 14, color: Colors.black)),
+                          subtitle: Text(widget.tile.name,
+                              style: TextStyle(
+                                  fontSize: 12, color: Color(0xff7C7C7C))),
+                          trailing: Wrap(
+                            children: [
+                              Icon(Icons.keyboard_arrow_right,
+                                  size: 35.0, color: Colors.black54),
+                            ],
+                          ),
+                          onTap: () => {
+                            Navigator.push(
+                                context,
+                                MaterialPageRoute(
+                                    builder: (context) => TileName(
+                                          tile: widget.tile,
+                                        )))
+                          }, // Debug: change to edit
+                        ),
+                      ],
+                    ),
+                  ),
+                  Container(
+                    decoration: BoxDecoration(
+                      border: Border(
+                        bottom: new BorderSide(color: Colors.black12),
+                      ),
+                    ),
```

Comment:
Ohh, this looks like a divider - @nganphan123 also have a really cool `Divider` component, so you might be able to pull from her repo to retrieve that component.

Reply:
I think I will do this in the next PR since retrieving from Jill's branch now can create some unexpected files since her PR hasn't been merged to master yet